shaders/src/common_types.fs

```diff
@@ -1,4 +1,4 @@
-#if !defined(TARGET_MOBILE) || defined(TARGET_LANGUAGE_SPIRV)
+#if defined(FILAMENT_SHADER_MODEL_GL41) || defined(FILAMENT_VULKAN_SEMANTICS)
 #define LAYOUT_LOCATION(x) layout(location = x)
```

Comment:
Do we really need the `layout(location = x)` for OpenGL desktop?

Reply:
looks like not!